docs/testrunner-toolkit.md

```diff
@@ -156,6 +157,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Puppeteer Version|Supported Browsers|
 |-----|----|
+|9.1.1|Please see [release notes](https://github.com/saucelabs/sauce-puppeteer-runner/releases/tag/v1.1.0)|
```

Comment:
Same check on the runner version vs. the fw version. :)

Reply:
They're not matching each other. so it's correct :)